src/SearchInput/SearchInput.tsx

```diff
@@ -136,7 +136,13 @@ export const SearchInput = forwardRef<HTMLInputElement, SearchInputProps>(
             onChange={handleInputChange}
             outlined={outlined}
             selected={isSelected}
-            onBlur={onBlur}
+            onBlur={(e) => {
+              if (displayedInputText === '') {
```

Comment:
This sets the value back to selectedValue in case the user deleted the input and clicked away